docs/openapi/consolidatore-callback-v1.yml

```diff
@@ -0,0 +1,130 @@
+        degli errori all'interno del processamento delle macchine a stati.<br/>
+        Il consolidatore deve validare la sintatticamente la richiesta e, se tale 
+        validazione ha successo, registrare la richiesta in maniera sincrona per poi trasmetterla al recapitista 
+        utilizzando l'apposita API
+      parameters:
+        - $ref: '#/components/parameters/xPagopaExtchServiceId'
+        - $ref: '#/components/parameters/xPagopaExtchApiKey'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: array
+              minItems: 1
+              maxItems: 250
+              items:
+                $ref: '#/components/schemas/PaperProgressFailure'
+        required: true
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              example: { resultCode: '200.00', resultDescription: 'OK', clientResponseTimeStamp: '2019-08-24T14:15:22Z'}
+              schema:
+                $ref: '#/components/schemas/OperationResultCodeResponse'
+        '400':
+          description: errore di validazione della richiesta
+          content:
+            application/json:
+              examples: 
+                syntaxError: 
+                  summary: 'Errore sintattico'
+                  value: { resultCode: '400.01', resultDescription: 'Syntax Error', errorList: [ 'field requestId is required'], clientResponseTimeStamp: '2019-08-24T14:15:22Z'}
+                semanticError: 
+                  summary: 'Errore semantico'
+                  value: { resultCode: '400.02', resultDescription: 'Semantic Error', errorList: [ 'unrecognized product Type'], clientResponseTimeStamp: '2019-08-24T14:15:22Z'}
+              schema:
+                $ref: '#/components/schemas/OperationResultCodeResponse'
+        '401':
+          description: Mancano le credenziali di identificazione o sono sbagliate
+          content:
+            application/json:
+              example: { resultCode: '401.00', resultDescription: 'Authentication Failed', clientResponseTimeStamp: '2019-08-24T14:15:22Z'}
+              schema:
+                $ref: '#/components/schemas/OperationResultCodeResponse'
+  
+
+components:
+
+  parameters:
+
+    xPagopaExtchServiceId:
+      $ref: 'schemas-paper-v2.yaml#/components/parameters/xPagopaExtchServiceId'
+    
+    xPagopaExtchApiKey:
+      $ref: 'schemas-paper-v2.yaml#/components/parameters/xPagopaExtchApiKey'
+  
+  schemas:
+
+    PaperProgressFailure:
+      required:
+        - requestId
+        - discardCode
+        - discardDescription
+        - discardTimeStamp
+      type: object
+      properties:
+        requestId:
+          type: string
+          minLength: 30
+          maxLength: 250
+          example: 'ABCD-HILM-YKWX-202202-1_rec0_try1'
+          description: >-
+            Identificativo della richiesta.
+        failureTriggeredBy:
+          description: >-
```

Comment:
Su "description" Intellij mi dice: "Schema validation: Property 'description' is not allowed". Credo perché si usi la description dell'oggetto ref.